docs/docs/experiments/experiments.md

```diff
@@ -21,13 +21,19 @@ are intended to replace.
 
 You can enable an experimental feature by:
 
-1. Using the `--x-<feature>` flag. This is intended for one-off invocations of
-   Task to test out experimental features. You can also disable a feature by
-   specifying a falsy value such as `--x-<feature>=false`.
-1. Using the `TASK_X_<FEATURE>=1` environment variable. This is intended for
-   permanently enabling experimental features in your environment.
-
-Flags will always override environment variables.
+1. Using the relevant environment variable in front of a task command. For
+   example, `TASK_X_<FEATURE>=1 task <my-task>`. This is intended for one-off
```

Comment:
I just pushed a bug fix to `main` that caused a conflict on this PR.

I replaced `<VAR>` occurrences with `{VAR}` because Crowdin sometimes tries to parse `<foo>` as a HTML tag, making the Docusaurus build fail.